diagnostic.rb

```diff
@@ -1,9 +1,54 @@
 # Question 1. Paste your code below.
 
+class CreatePets < ActiveRecord::Migration
+  def change
+    create_table :pets do |t|
+      t.references :people, index: true, foreign_key: true
+
+      t.timestamps null: false
+    end
+  end
+end
+
 # Question 2. Paste your code below.
 
+class Person < ActiveRecord::Base
+  has_many :pets, inverse_of: :pets, class_name: 'Person'
+end
+
+class Pet < ActiveRecord::Base
+  belongs_to :people, inverse_of: :owner, foreign_key: 'owner_id'
```

Comment:
`belongs_to :people` should be `belongs_to :person`, And you dont need an invers_of in the Person class nor do you need to specify class_name or foreign_key